src/main/java/eu/yals/controllers/rest/StoreRestController.java

```diff
@@ -25,32 +25,46 @@
 import java.util.HashSet;
 import java.util.Set;
 
+import static eu.yals.constants.HttpCode.*;
+
 /**
- * Stores long link to storage
+ * Stores long link to storage.
  *
  * @since 1.0
  */
 @Slf4j
 @RestController
 public class StoreRestController {
-    private static final String TAG = "[API Store]";
+    private static final String TAG = "[" + StoreRestController.class.getSimpleName() + "]";
 
     private final LinkService linkService;
 
-    public StoreRestController(LinkService linkService) {
-        this.linkService = linkService;
+    /**
+     * Constructor for Spring autowiring.
+     *
+     * @param linksService service which stores links to DB
+     */
+    public StoreRestController(final LinkService linksService) {
+        this.linkService = linksService;
     }
 
+    /**
+     * API Endpoint to store link into DB.
+     *
+     * @param body     string with body of HTTP request
+     * @param response response
+     * @return json with reply
+     */
     @RequestMapping(method = {RequestMethod.POST, RequestMethod.PUT},
             value = Endpoint.Api.STORE_API)
-    public Json store(@RequestBody String body, HttpServletResponse response) {
+    public Json store(final @RequestBody String body, final HttpServletResponse response) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'store(String, HttpServletResponse)' has an NPath complexity of 768](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)